### PR TITLE
Standard caseto camel case publish yml file

### PR DESCRIPTION
**Pull request by MaheshNVKA**

### CODE REVIEW

File: src/Kmd.Logic.Gateway.Automation/ValidatePublishing.cs [hunk 1]
```diff
@@ -30,7 +31,8 @@ public ValidatePublishing(HttpClient httpClient, LogicTokenProviderFactory token
         public async Task<ValidationResult> ValidateAsync(string folderPath)
         {
             var publishYml = File.ReadAllText(Path.Combine(folderPath, "publish.yml"));
-            var publishFileModel = new Deserializer().Deserialize<PublishFileModel>(publishYml);
+
```
Comment (NakulSeerpu) [inline src/Kmd.Logic.Gateway.Automation/ValidatePublishing.cs:34]: Remove the extra line

Comment (MaheshNVKA) [reply src/Kmd.Logic.Gateway.Automation/ValidatePublishing.cs:34]: Okay i will remove the extra line

**Review by Araballi — APPROVED** [review]
LGTM

File: src/Kmd.Logic.Gateway.Automation/ValidatePublishing.cs (hunk 1 above)
Comment (MaheshNVKA) [reply src/Kmd.Logic.Gateway.Automation/ValidatePublishing.cs]: I did changes removed the line as specified.

**Comment by mlatoszek** [comment]
LGTM

**Review by mlatoszek — APPROVED** [review]

**Pull request merged**